src/langs/en.json

```diff
@@ -64,5 +65,8 @@
   "NO_ITEMS_EXIT": "No items found.",
   "NO_ITEMS_MATCH_SEARCH": "No search results found.",
   "HOME_SHOW_MORE": "Show more...",
-  "ACTIONS_MIGHT_HAS_NO_GEOLOCATION": "Some actions might have no geolocation records"
+  "ACTIONS_MIGHT_HAS_NO_GEOLOCATION": "Some actions might have no geolocation records",
+  "EXPORT_ANALYTICS_TITLE": "Export Analytics",
+  "EXPORT_ANALYTICS_DESCRIPTION": "You can export the analytical traces left by users while interacting with your item (and its children). It may take some time to gather the data into an archive, after which you'll receive a link via email for download. This link remains valid for 7 days.\n \n You're allowed one data request per day. Any additional requests will yield the same archive. For instance, if an archive is generated at 8 AM and you request another export at 10 AM, you'll receive the same archive as the one generated at 8 AM, without including the data between 8 AM and 10 AM.",
```

Comment:
```suggestion
  "EXPORT_ANALYTICS_DESCRIPTION": "You can export the analytical traces left by users while interacting with your item (and its children). It may take some time to gather the data into an archive, after which you'll receive a link via email for download. This link remains valid for 7 days.\n \n You are allowed one data request per day. Any additional requests will yield the same archive. For instance, if an archive is generated at 8 AM and you request another export at 10 AM, you'll receive the same archive as the one generated at 8 AM, without including the data between 8 AM and 10 AM.",
```